thoth-api/src/graphql/model.rs

```diff
@@ -3613,3 +3616,41 @@ fn user_can_edit_publication(publication_id: Uuid, context: &Context) -> Result<
         .expect("Error checking permissions");
     context.account_access.can_edit(pub_id)
 }
+
+fn issue_imprints_match(work_id: Uuid, series_id: Uuid, context: &Context) -> Result<()> {
+    let series_imprint = crate::schema::series::table
+        .select(crate::schema::series::imprint_id)
+        .filter(crate::schema::series::series_id.eq(series_id))
+        .first::<Uuid>(&context.db.get().unwrap())
+        .expect("Error loading series for issue");
+    let work_imprint = crate::schema::work::table
+        .select(crate::schema::work::imprint_id)
+        .filter(crate::schema::work::work_id.eq(work_id))
+        .first::<Uuid>(&context.db.get().unwrap())
+        .expect("Error loading work for issue");
```

Comment:
These two queries could be combined into a single query using the syntax `select * from (select work.imprint_id from work where work.work_id = [x]) as a, (select series.imprint_id from series where series.series_id = [y]) as b;`, but I'm not sure if Diesel supports it (or if it's desirable - I think it might be slower than doing separate queries).

Reply:
It's probably better to leave it as it is so it can be more legible. It won't affect speed